src/analytics/api/utils/pre_request.py

```diff
@@ -1,7 +1,7 @@
 from itertools import chain
 import re
 from flask import request, jsonify, make_response
-from config import BASE_URL
+from config import API_V2_BASE_URL
```

Comment:
Hi @NoahNsimbe , thanks for this. Much appreciated!

I am just curious as to where the base URL for v1 is defined from.
In the PR, I notice that we replace v1 with V2, so what happens to the BASE_URL for v1?

Reply:
this is only used for swagger. Let me add v1 to swagger. The actual apis are configured [here](https://github.com/airqo-platform/AirQo-api/pull/1524#discussion_r1183009281)

Reply:
Alright, please do so in this PR. It might be hard to do it afterwards. @NoahNsimbe